android/app/src/main/java/org/uwaterloo/subletr/pages/home/list/components/BasicFilterLayout.kt

```diff
@@ -204,7 +200,7 @@ fun DefaultFilterButton(
 			horizontal = dimensionResource(id = R.dimen.s),
 			vertical = dimensionResource(id = R.dimen.xs),
 		),
-		content = { Text(text = text, color = if (isSelected) Color.White else Color.Black) },
+		content = { Text(text = text, color = if (isSelected) Color.White else MaterialTheme.subletrPalette.primaryTextColor) },
```

Comment:
Shouldn't we use a MaterialTheme color for the isSelected case as well? Not sure, just making sure we considered that as well :)

Reply:
It looks nice with white color in both modes, so probably don't need to change it.